src/styles/page.scss

```diff
@@ -0,0 +1,17 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  color: $main-accent;
```

Comment:
Ensure that the SCSS variable `$main-accent` is defined in your stylesheets. If it's not defined, it will cause an error during the SCSS compilation.